examples/experimental/tongye/opacus_with_Plan/Opacus_Syft_Data_Scientist.ipynb

```diff
@@ -0,0 +1,627 @@
+{
```

Comment:
<p>The super long duration might be an artifact of passing the dataset length instead of the dataloader length: in this case, batch size is 64 so there's going to be 64x the batches :D </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/5333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>yeah, we will see after fix the dataset length typo</p>